ext/bcmath/php_bcmath.h

```diff
@@ -19,6 +19,8 @@
 
 #include "libbcmath/src/bcmath.h"
 #include "zend_API.h"
+#include "php.h"
```

Comment:
I want to read and use round constants from php_math.h, but I need to resolve `PHPAPI`.
If you have any good ideas, it would be helpful if you could let me know.

Reply:
Ahhh right it's that issue... one "dumb" solution is to redefine the constants for BCMATH and expose them and have a PHPT test to check that the values are really equal to each other.

Reply:
I've been thinking about this for a while now, how if I separate out just the standard round-mode constants into a file like "php_math_round.h" and include that?